package.json

```diff
@@ -78,21 +79,20 @@
 		"eslint-config-next": "^13.4.12",
 		"eslint-config-prettier": "^8.8.0",
 		"facepaint": "^1.2.1",
-		"html-validate": "^8.1.0",
+		"html-validate": "^8.27.0",
 		"jest": "^29.7.0",
 		"jest-axe": "^8.0.0",
 		"jest-environment-jsdom": "^29.7.0",
-		"next-router-mock": "^0.9.9",
```

Comment:
We don't need this for Storybook anymore?

Reply:
It doesn't work with `@storybook/next` 🤷